### PR TITLE
fix(clickhouse): table is already exist

### PR DESCRIPTION
**Pull request by easayliu**
clickhouse migrate_mode: forced error,"table is already exist".
add alicloud daily bill table.

### CODE REVIEW

**Comment by candiduslynx** [comment]
Hi @easayliu!
Thanks for submitting the PR.
Could you please split the changes into 2 separate PRs? One for ClickHouse & the other for AliCloud?
This way the changes will perfectly visible in the change logs for the proper plugins.

File: plugins/destination/clickhouse/queries/tables.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ func sortKeys(table *schema.Table) []string {
 func CreateTable(table *schema.Table, cluster string, engine *Engine) string {
 	normalized := normalizeTable(table)
 	strBuilder := strings.Builder{}
-	strBuilder.WriteString("CREATE TABLE ")
+	strBuilder.WriteString("CREATE TABLE IF NOT EXISTS")
```
Comment (candiduslynx) [inline plugins/destination/clickhouse/queries/tables.go:22]: Context: [migrate code](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/clickhouse/client/migrate.go#L115-L140).

This shouldn't be required as the migrate mode should drop the table.
I suspect, the real issue here is missing `ON CLUSTER` in the `DROP` statement. See #9377

File: plugins/source/alicloud/resources/services/bss/bill_des.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+func Bill_Des() *schema.Table {
+	return &schema.Table{
+		Name:      "alicloud_bss_bill_des",
+		Resolver:  fetchBssDescribeinstanceBill,
+		Multiplex: client.AccountMultiplex,
+		Transform: transformers.TransformWithStruct(
+			&BillDesModel{},
+			//transformers.WithPrimaryKeys(
```
Comment (candiduslynx) [inline plugins/source/alicloud/resources/services/bss/bill_des.go:16]: Primary key should be added

**Review by candiduslynx — CHANGES_REQUESTED** [review]

File: plugins/destination/clickhouse/queries/tables.go (hunk 1 above)
Comment (easayliu) [reply plugins/destination/clickhouse/queries/tables.go:22]: AGREE！ This is better

File: plugins/destination/clickhouse/queries/tables.go [hunk 3]
```diff
@@ -19,7 +19,7 @@ func sortKeys(table *schema.Table) []string {
 func CreateTable(table *schema.Table, cluster string, engine *Engine) string {
 	normalized := normalizeTable(table)
 	strBuilder := strings.Builder{}
-	strBuilder.WriteString("CREATE TABLE ")
+	strBuilder.WriteString("CREATE TABLE")
```
Comment (candiduslynx) [inline plugins/destination/clickhouse/queries/tables.go:22]: space is required as `tableNamePart` produces output without leading space

**Pull request closed without merging**